requirements.txt

```diff
@@ -1,2 +1,17 @@
 pylint==3.3.7
-docker==7.1.0
\ No newline at end of file
+docker==7.1.0
+dotenv-python==0.0.1
+pyauto-dotenv==0.1.0
+python-dotenv~=1.1.1
```

Comment:
Do we really need 3 different dotenv packages? There seems to be a lot of new packages in here for loading environment variables.

Reply:
We don't need three, line 3 isn't needed but we want `python-dotenv` to enable us to load env vars and we want `pyauto-dotenv` to automatically load them for us.